### ISSUE TITLE
Scroll with ElasticSearch 1.4.2

### ISSUE THREAD

**Issue by cspiegl** [elasticsearch/elasticsearch-js#174]
I noticed that the scrolling seams to be broken with version 1.4.2 of elastic search.

If I pas the scroll_id via the body (see: https://github.com/elasticsearch/elasticsearch-js/issues/113#issuecomment-46214315) it works. If I set it as `ec.scroll({scrollId: response._scroll_id, scroll: '30s'}, getMoreUntilDone)` it does not work and fails with the following error:

```
Elasticsearch ERROR: 2015-01-06T15:45:01Z
  Error: Request error, retrying -- socket hang up
    at Log.error (/Users/cspiegl/Amazine/ThesisAnalyzeLog/node_modules/elasticsearch/src/lib/log.js:213:60)
    at checkRespForFailure (/Users/cspiegl/Amazine/ThesisAnalyzeLog/node_modules/elasticsearch/src/lib/transport.js:195:18)
    at HttpConnector.<anonymous> (/Users/cspiegl/Amazine/ThesisAnalyzeLog/node_modules/elasticsearch/src/lib/connectors/http.js:154:7)
    at ClientRequest.bound (/Users/cspiegl/Amazine/ThesisAnalyzeLog/node_modules/elasticsearch/node_modules/lodash-node/modern/internals/baseBind.js:56:17)
    at ClientRequest.emit (events.js:107:17)
    at Socket.socketOnEnd (_http_client.js:292:9)
    at Socket.emit (events.js:129:20)
    at _stream_readable.js:907:16
    at process._tickCallback (node.js:372:11)


Elasticsearch WARNING: 2015-01-06T15:45:01Z
  Unable to revive connection: http://127.0.0.1:8091/

Elasticsearch WARNING: 2015-01-06T15:45:01Z
  No living connections
```

**Comment by spenceralger** [comment]
This is something that people have regularly run into, and it appears it is something that the [python client](https://github.com/elasticsearch/elasticsearch-py/blob/d3e745ab773be42ecc7914405721aebf56a6a766/elasticsearch/client/__init__.py#L617) seems to handle gracefully. I pushed [some commits](https://github.com/elasticsearch/elasticsearch-js/compare/2052921...master) to master that I'll release soon. These changes will send the scrollId as the request body as long as no other body was specified.

Hope that helps!

**Comment by spenceralger** [comment]
Released 3.1.1, thanks!
